### DISCUSSION TITLE
netcore issue bouncycastle

### DISCUSSION THREAD

**Discussion by SimpleSimon1** [bcgit/bc-csharp#510] [General]
![image](https://github.com/bcgit/bc-csharp/assets/148480658/03aee508-23ea-41f1-aba5-f9d63ceb0c14)
![image](https://github.com/bcgit/bc-csharp/assets/148480658/62d841d1-029b-46d5-a43b-b4c698272d8b)

I am getting this when i try to run a build in visual studio is there a fix for this

**Comment by SimpleSimon1** [comment]
I have tried downloading this bouncycaslte thing from places however still getting the same issue

**Comment by peterdettman** [comment]
Our official NuGet package is here: https://www.nuget.org/packages/BouncyCastle.Cryptography .
